pkgs/by-name/co/copilot-language-server/package.nix

```diff
@@ -1,31 +1,55 @@
 {
   lib,
-  buildNpmPackage,
+  stdenv,
   fetchurl,
+  unzip,
+  nix-update-script,
 }:
 
-buildNpmPackage rec {
+let
+  arch =
+    {
+      aarch64-darwin = "arm64";
+      aarch64-linux = "arm64";
+      x86_64-darwin = "x64";
+      x86_64-linux = "x64";
+    }
+    ."${stdenv.hostPlatform.system}" or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+  os =
+    {
+      aarch64-darwin = "darwin";
+      aarch64-linux = "linux";
+      x86_64-darwin = "darwin";
+      x86_64-linux = "linux";
+    }
+    ."${stdenv.hostPlatform.system}" or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+in
+
+stdenv.mkDerivation (finalAttrs: {
   pname = "copilot-language-server";
   version = "1.275.0";
 
   src = fetchurl {
```

Comment:
How about using `fetchzip` and removing the `unzip` requirement?

Reply:
Done.